pandas/tests/arrays/test_datetimelike.py

```diff
@@ -126,6 +126,24 @@ def test_int_properties(self, datetime_index, propname):
 
         tm.assert_numpy_array_equal(result, expected)
 
+    def test_array(self, datetime_index):
```

Comment:
while you're at it, can you make sure there are analogous tests for the DTI/TDI/PI index classes?

Reply:
Do you know if there are any existing tests on this somewhere (for the index classes). Not directly finding something